src/util/swapUtils.ts

```diff
@@ -747,7 +748,7 @@ export class SwapUtils {
     const accountDetails = walletState.currentLoggedInWallet.accounts.find((account) => account.address == accAddress.plain());
     const encryptedPassword = WalletUtils.createPassword(walletPassword);
     let privateKey = WalletUtils.decryptPrivateKey(encryptedPassword, accountDetails.encrypted, accountDetails.iv);
-    const account = Account.createFromPrivateKey(privateKey, ChainUtils.getNetworkType(networkState.currentNetworkProfile.network.type));
+    const account = Account.createFromPrivateKey(privateKey, ChainUtils.getNetworkType(AppState.networkType));
```

Comment:
just use AppState.networkType directly, no need to use ChainUtils.getNetworkType